.github/workflows/go-test.yml

```
@@ -10,7 +10,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu", "windows", "macos" ]
-        go: ["1.19.x","1.20.x"]
+        go: ["1.20.x"]
```

Why ?
```suggestion
        go: ["1.19.x", "1.20.x"]
```
We usually support the two latest golang releases.

---

Likely because of that CI fail: https://github.com/ipfs/iptb-plugins/actions/runs/4992996363

---

Can you try downgrading multierr ?
Kubo doesn't correctly work with `1.20` yet, we don't want to drop 0.19 and be cornered.